addons/mail/static/src/components/composer/composer.xml

```diff
@@ -84,9 +84,9 @@
                         composerViewLocalId="composerView.localId"
                         hasMentionSuggestionsBelowPosition="props.hasMentionSuggestionsBelowPosition"
                         isCompact="props.isCompact"
-                        sendShortcuts="props.textInputSendShortcuts"
                         onComposerTextInputSendShortcut="_onComposerTextInputSendShortcut"
                         onPaste="_onPasteTextInput"
+                        t-on-o-composer-text-input-send-shortcut="_onComposerTextInputSendShortcut"
```

Comment:
where is there still an event here?

Reply:
Should have been removed, residue from conflicts during rebase